src/view/LoginView.java

```diff
@@ -59,6 +68,16 @@ public void actionPerformed(ActionEvent evt) {
 
     cancel.addActionListener(this);
 
+    switchToSignup.addActionListener(
+        new ActionListener() {
+          @Override
+          public void actionPerformed(ActionEvent e) {
+            switchViewController.switchTo("sign up");
```

Comment:
I don't like this string, but we'll fix it once we've merged #6

Reply:
sure